packages-exp/auth-exp/src/core/auth/register.ts

```diff
@@ -24,13 +24,16 @@ import { AuthErrorCode } from '../errors';
 import { assert } from '../util/assert';
 import { _getClientVersion, ClientPlatform } from '../util/version';
 import {
+  _castAuth,
   AuthImpl,
   DEFAULT_API_HOST,
   DEFAULT_API_SCHEME,
   DEFAULT_TOKEN_API_HOST
 } from './auth_impl';
+import { AuthInternal } from './firebase_internal';
 
 export const _AUTH_COMPONENT_NAME = 'auth-exp';
+export const _AUTH_INTERNAL_COMPONENT_NAME = 'auth-internal-exp';
```

Comment:
is this right? or should it be auth-interop?

Reply:
@Feiyang1 for guidance. I believe internal is correct since that's what we do now, but maybe we're changing our language?
https://github.com/firebase/firebase-js-sdk/search?q=%27auth-internal%27&unscoped_q=%27auth-internal%27

Reply:
`auth-interna` is correct